h/features/test/models_test.py

```diff
@@ -71,3 +73,35 @@ def features_pending_removal_override(self, request):
         }, clear=True)
         patcher.start()
         request.addfinalizer(patcher.stop)
+
+
+class TestCohort(object):
+    def test_init(self):
+        name = "My Hypothesis Cohort"
+
+        cohort = FeatureCohort(name=name)
+        db.Session.add(cohort)
+        db.Session.flush()
+
+        assert cohort.id
+        assert cohort.name == name
+        assert cohort.created
+        assert cohort.updated
+
+    def test_get_by_id_when_id_does_exist(self):
```

Comment:
Honestly, this test and the next one are just testing sqlalchemy. We don't need to test that sqlalchemy works.